pages/container/fit-height.page.tsx

```diff
@@ -101,7 +109,89 @@ export default function () {
         <div className={styles.heightLimit}>
           <LargeContainer />
         </div>
+        <h2>Content variants</h2>
```

Comment:
Is it really suitable for this page?

If there is any height diff in the content above, everything below will be shifted, making this screenshot diff impossible to review

Let's stick to the principle, one use case – one page